lib/dunmanifestin/phrase.rb

```diff
@@ -74,17 +74,24 @@ def initialize dsl_string
 
     self.inflections_delegated_to_me = []
     self.demanded_inflections        = []
+    self.constraints                 = []
 
     components.each_with_index do |v, k|
-      inflections_delegated_to_me << v.to_sym if components[k-1] == '#'
-      demanded_inflections << v.to_sym if components[k-1] == '.'
+      case components[k-1]
+      when '#'
```

Comment:
Hm, actually, I never understood the difference between `#` and `.`, and I think I rather mutilated your meticulously constructed fantasy `|weapons` palette as a result. I had better review `inflections_delegated_to_me` and `demanded_inflections`.

Reply:
Check out `phrase_spec.rb`.